### ISSUE TITLE
Implementing deep CNN in Python using TensorFlow and Keras for face mask detection

### ISSUE THREAD

**Issue by jumapaul** [section-engineering-education/engineering-education#6736]
### Topic Suggestion
Implementing deep CNN in Python using TensorFlow and Keras for face mask detection

### Writing sample(s): 
https://www.section.io/engineering-education/eigen-decomposition-orthogonolization-and-orthonormalization-in-matlab/

#### Introduction
Deep CNN is implemented in Python using TensorFlow and Keras libraries. TensorFlow is an end-to-end open-source platform for machine learning created by the google brain team. TensorFlow has a comprehensive, flexible ecosystem of tools, libraries and community resources. It helps researchers to develop state-of-the-art machine learning applications.

In this tutorial, we show how a deep Convolutional Neural Network (CNN) can be implemented in Python using TensorFlow(TF) and Keras (K). We also cover its working. An interesting example is taken for implementation, where we can classify face images 'With Mask' and 'Without Mask'. The objective here is to detect face masks in offline mode.

#### Key takeaways 
- Proposed Deep CNN based scheme.
- Introduction to TensorFlow, Keras and Python.
- Image Dataset used.
- Python Code to implement deep CNN using TF and K.
- Simulation result analysis.

#### Article quality
Here we use the tensor flow and Keras libraries to implement the face mask detection using python.

**Comment by louisefindlay23** [comment]
Thank you for submitting your topic suggestion. There seems to be [many articles](https://www.google.com/search?client=firefox-b-d&q=Implementing+deep+CNN+in+Python+using+TensorFlow+and+Keras+for+face+mask+detection) that already cover this topic. Can you tell me how yours will build upon them and be unique to add value to our readers?

**Comment by jumapaul** [comment]
@louisefindlay23 Here, we investigate optimal parameter values for the Convolutional Neural Network model (CNN) in order to identify the existence of masks accurately without generating over-fitting. This technique attains excellent accuracy.

**Comment by louisefindlay23** [comment]
@jumapaul, are optimal parameter values your unique piece of content that has not been covered elsewhere?

**Comment by jumapaul** [comment]
@louisefindlay23 yes, We investigate these parameters and use them to improve the accuracy. This has not been covered elsewhere. Thank you.
